api/nakadi-event-bus-api.yaml

```diff
@@ -150,11 +178,11 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
         '404':
-          description: Not found. Typically in case of requesting from a non existing topic of partition.
+          description: Not found. Typically in case of requesting from a non existing topic or partition. Details are provided on the returned `Problem`.
           schema:
             $ref: '#/definitions/Problem'
         '500':
```

Comment:
I know that it wasn't added by you, but maybe we should not define 500 in our API. For me throwing 500 means that there is an error in your app. 503 seems to be more appropriate code.